base/missing.jl

```diff
@@ -107,16 +107,19 @@ max(::Missing, ::Any)     = missing
 max(::Any,     ::Missing) = missing
 
 # Rounding and related functions
-for f in (:(ceil), :(floor), :(round), :(trunc))
+round(::Missing, ::RoundingMode=RoundNearest; sigdigits::Integer=0, digits::Integer=0, base::Integer=0) = missing
+round(::Type{>:Missing}, ::Missing, ::RoundingMode=RoundNearest) = missing
+round(::Type{T}, ::Missing, ::RoundingMode=RoundNearest) where {T} =
+    throw(MissingException("cannot convert a missing value to type $T: use Union{$T, Missing} instead"))
+round(::Type{T}, x::Any, ::RoundingMode=RoundNearest) where {T>:Missing} = round(nonmissingtype(T), x)
+# to fix ambiguities
+round(::Type{T}, x::Rational, ::RoundingMode=RoundNearest) where {T>:Missing} = round(nonmissingtype(T), x)
+round(::Type{T}, x::Rational{Bool}, ::RoundingMode=RoundNearest) where {T>:Missing} = round(nonmissingtype(T), x)
+
+for f in (:(ceil), :(floor), :(trunc))
     @eval begin
-        ($f)(::Missing, digits::Integer=0, base::Integer=0) = missing
-        ($f)(::Type{>:Missing}, ::Missing) = missing
-        ($f)(::Type{T}, ::Missing) where {T} =
-            throw(MissingException("cannot convert a missing value to type $T: use Union{$T, Missing} instead"))
-        ($f)(::Type{T}, x::Any) where {T>:Missing} = $f(nonmissingtype(T), x)
```

Comment:
Where have these functions gone?

Reply:
Good point.

Reply:
I thought I had taken care of them with the wrapper that calls round(), but this didn't take care of all the possible cases. Tests now pass again for me.